waterloop-site/src/components/SustainableTech/Teams/TeamsCarousel.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { useSwipeable } from 'react-swipeable';
+import styled from 'styled-components';
+import useTeamImages from './hooks/team-images';
+
+const TeamsContentWrapper = styled.div`
+  max-width: 1200px;
+  margin: 50px 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  flex-direction: column;
+`;
+
+const TeamCardContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 30px;
+  padding: 40px 20px;
+  background-color: #618A4D;
+  box-shadow: 0px 4px 40px rgba(0, 0, 0, 0.15);
+  border-radius: 8px;
+  @media screen and (max-width: 768px) {
+    gap: 10px;
+    flex-direction: column;
+    padding: 40px;
+  }
+`;
+
+const SlideContainer = styled.div`
+  display: flex;
+  width: 100%;
+  gap: 30px;
+  justify-content: center;
+  align-items: center;
+  @media screen and (max-width: 768px) {
+    gap: 10px;
+    flex-direction: column;
+  }
+`;
+
+const TeamImageWrapper = styled.div`
+  max-width: 300px;
+`;
+
+const TeamImage = styled.img`
+  width: 100%;
+  border-radius: 8px;
+`;
+
+const TeamInfoHeading = styled.h2`
```

Comment:
I would re-check the typography. Currently, the text is italicized, whereas it should be non-italicized like in the Figma design.